packages/test-commons/lib/test-commons.js

```diff
@@ -1,6 +1,8 @@
 const { SchemaOperations } = require('velo-external-db-commons')
 const { AddColumn, RemoveColumn } = SchemaOperations
 
+const suitDef = (name, setup) => ( { name, setup } )
```

Comment:
not sure why this is needed here, why not just inline it ? or just write the config on the resources js at you want it ?